src/errors.ts

```diff
@@ -93,18 +94,16 @@ export class CustomError extends Error {
  * additional error context.
  *
  * @param e Error to log
- * @param message Text to add in the logger.error() message and top-level Sentry error message
- * @param extra Additional context to include in the log message (and `extra` field in Sentry)
- * @param sendTelemetry Whether to send the error to Sentry (default: `false`)
+ * @param messagePrefix Prefix to include in the logger.error() message
```

Comment:
Ah sorry, I missed one section of the merge conflict resolution that pulled in the old comment for `message` - could you please pull that new `message` param text over?